cms/templates/import.html

```diff
@@ -127,6 +191,19 @@ <h3 class="title">${_("Success")}</h3>
       </form>
     </article>
 
+  %if library:
+    <aside class="content-supplementary" role="complimentary">
+      <div class="bit">
+          <h3 class="title-3">${_("Why import a library?")}</h3>
+          <p>${_("You might want to update an existing library to a new version, or replace an existing library entirely. You might also have developed a library outside of Studio.")}</p>
+      </div>
+
+      <div class="bit">
+          <h3 class="title-3">${_("Warning: Importing while a library is in use")}</h3>
+          <p>${_("If you perform an import while your library is in use, any randomized content blocks which use it will need to be refreshed to take advantage of the new content.")}</p>
```

Comment:
Maybe a note instead of a warning: "Note: If you change and import a library that is referenced by randomized content blocks in one or more courses, those courses do not automatically use the updated content. You must manually refresh the randomized content blocks to bring them up to date with the latest library content."